src/handlers/mutation.ts

```diff
@@ -39,15 +40,19 @@ export async function executeMutation(
             properties: additionalProperties,
           });
         }
-        // todo: something with the response?
-        const insertResponse = await creator.do();
-
-        // console.log("insert response", JSON.stringify(insertResponse));
 
-        // todo: handle returning fields, based on insert response.
+        const insertResponse = await creator.do();
+        
```

Comment:
This is the only meaningful thing to check in this PR where the failure check is happening

Reply:
❓ Does it make sense to pass the error through the GDC? Doesn't need to be in this PR but Hasura might have some error handling we can take advantage of by returning a 4xx error from the GDC iteself.

Reply:
Yeah I thought about it but we are doing all sorts of thing on the BE and instead of handling all those cases (we are using a batch insert to insert a single item atm) returning only the affected rows and let the BE decide what it wants to do. I configured retries for e.g., here https://github.com/digs-com/app/pull/2568

Reply:
Certainly can handle that as you mentioned